pkg/referrerstore/oras/oras.go

```diff
@@ -58,7 +57,7 @@ const (
 	HttpMaxIdleConns                      = 100
 	HttpMaxConnsPerHost                   = 100
 	HttpMaxIdleConnsPerHost               = 100
-	HttpRetryMax                          = 10
+	HttpRetryMax                          = 5
 	HttpRetryDurationMin    time.Duration = 100 * time.Millisecond
 	HttpRetryDurationMax    time.Duration = 200 * time.Millisecond
```

Comment:
does this say retry every 200 ms ? wondering if this is too soon?

Reply:
Yeah at max it'll retry after 200ms. Now that we have 5 seconds, I think increasing it should be fine. Maybe like 2 seconds as the max retry amount?

Reply:
I wonder does the cache library implement exponential backoff...  if we have a HttpRetryDurationMax   of 1sec, in a duration of 5sec, we might get > 5 attempts... If its 2sec, will we get too few attempts? how many times do we want to retry?

Reply:
thanks for checking, sounds good